castai/add-castai-removal-disabled/add-castai-removal-disabled.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: add-castai-removal-disabled
+  annotations:
+    policies.kyverno.io/title: Add CAST AI Removal Disabled
+    policies.kyverno.io/category: CAST AI, EKS Best Practices
```

Comment:
I don't think this is an EKS Best Practices policy so I'd remove that.

Reply:
Got it, updated this and the ditto comments.